engine/server/engine/centralized_logs/client_implementations/kurtosis_backend/kurtosis_backend_logs_database_client_test.go

```diff
@@ -169,39 +169,39 @@ func TestStreamUserServiceLogs_FilteredAllLogLines(t *testing.T) {
 
 }
 
-func TestStreamUserServiceLogs_FilteredAllLogLinesWithFollowLogs(t *testing.T) {
-	expectedServiceAmountLogLinesByServiceUuid := map[service.ServiceUUID]int{
```

Comment:
The follow logs do not work at the moment because they will get timeout after 2 seconds; need to add a kill signal.. which i can we can in next pr.. @tedim52 .. this should make kurtosis service logs -f usable until then.